File: Backend/Models/Comment.cs
namespace Models
{
    using System;
    using System.Collections.Generic;
    using System.Linq;
    using System.Text;
    using System.Threading.Tasks;

    public class Comment
    {
        public string CommentId { get; set; }

        public string UserId { get; set; }

        public ApplicationUser User { get; set; }
        
        public string? CommentText { get; set; }

        public int? Qualification { get; set; }

        public DateTime CommentDate { get; set; }

        public string MovieId { get; set; }

        public Movie? Movies { get; set; }

    }
}

File: Frontend/src/domain/Movie/Comment.ts
import { CommentDTO } from "../dto/CommentDTO";

export class Comment {
    public commentId: string;
    public userId: string;
    public commentText: string | null;
    public qualification: number | null;
    public commentDate: Date;
  
    constructor(commentData: CommentDTO) {
      this.commentId = commentData.commentId;
      this.userId = commentData.userId;
      this.commentText = commentData.commentText ?? null;
      this.qualification = commentData.qualification ?? null;
      this.commentDate = new Date(commentData.commentDate);
    }
  }